backend/src/packages/chaiNNer_pytorch/pytorch/processing/inpaint.py

```diff
@@ -51,22 +51,30 @@ def pad_img_to_modulo(
 def inpaint(
     img: np.ndarray,
     mask: np.ndarray,
-    model: PyTorchInpaintModel,
+    model: InpaintModelDescriptor,
     options: PyTorchSettings,
 ):
     with torch.no_grad():
         # TODO: use bfloat16 if RTX
-        use_fp16 = options.use_fp16 and model.supports_fp16
+        use_fp16 = options.use_fp16 and model.supports_half
         device = options.device
 
         model = model.to(device)
-        model = model.half() if use_fp16 else model.float()
+        model.model.half() if use_fp16 else model.model.float()
 
         orig_height, orig_width, _ = get_h_w_c(img)
 
-        img = pad_img_to_modulo(img, model.pad_mod, model.pad_to_square, model.min_size)
+        img = pad_img_to_modulo(
+            img,
+            model.size_requirements.multiple_of or 0,
```

Comment:
Multiple of 0...

Yeah, so 0 will cause a divide by zero error. Use `or 1`. Same below.

Reply:
Whoops 😅